### PR TITLE
[bitnami/opencart] Chart standardized

### PR DESCRIPTION
**Pull request by migruiz4**
**Description of the change**

This PR standardizes the bitnami/opencart chart to be in line with the rest of the catalog.

**Checklist**

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

**Review by alvneiayu — APPROVED** [review]
LGTM

File: bitnami/opencart/values.yaml [hunk 1]
```diff
@@ -186,9 +198,10 @@ persistence:
   ## Requires persistence.enabled: true
   ## If defined, PVC must be created manually before volume will be bound
   ##
-  ## @param persistence.accessMode PVC Access Mode for OpenCart volume
+  ## @param persistence.accessModes [array] PVC Access Mode for OpenCart volume
```
Comment (miguelaeh) [inline bitnami/opencart/values.yaml:201]: Remove the `[array]` modifier, now arrays are properly parsed

File: bitnami/opencart/values.yaml [hunk 2]
```diff
@@ -242,11 +258,13 @@ nodeSelector: {}
 ## Configure resource requests and limits
 ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
 ## @param resources.requests [object] The requested resources for the container
+## @param resources.limits [object] The resources limits for the container
```
Comment (miguelaeh) [inline bitnami/opencart/values.yaml:261]:
```suggestion
## @param resources.limits The resources limits for the container
```

**Review by miguelaeh — CHANGES_REQUESTED** [review]
Re-check the metadata, modifiers are not needed except for special cases

File: bitnami/opencart/values.yaml [hunk 3]
```diff
@@ -348,15 +382,42 @@ service:
...
   ## @param service.sessionAffinity Control where client requests go, to the same pod or round-robin
   ## Values: ClientIP or None
   ## ref: https://kubernetes.io/docs/user-guide/services/
   ##
   sessionAffinity: "None"
+  ## @param service.sessionAffinityConfig Additional settings for the sessionAffinity
+  ## sessionAffinityConfig:
+  ##   clientIP:
+  ##     timeoutSeconds: 300
+  sessionAffinityConfig: {}
```
Comment (carrodher) [inline bitnami/opencart/values.yaml:420]:
```suggestion
  ##     timeoutSeconds: 300
  ##
  sessionAffinityConfig: {}
```

**Review by carrodher — APPROVED** [review]
LGTM

**Pull request merged**